src/Symfony/Component/Yaml/.yamllint.yml.dist

```diff
@@ -0,0 +1,8 @@
+yaml-lint:
```

Comment:
This file is no longer needed, is it?

Reply:
Yes I think we can drop the file, an example config is documented.